webview/src/setup/components/CliUnavailable.tsx

```diff
@@ -1,16 +1,89 @@
 import React from 'react'
+import { Button } from '../../shared/components/button/Button'
 import { EmptyState } from '../../shared/components/emptyState/EmptyState'
 
-export const CliUnavailable: React.FC = () => {
+export type CliUnavailableProps = {
+  installDvc: () => void
+  isPythonExtensionInstalled: boolean
+  pythonBinPath: string | undefined
+  selectPythonInterpreter: () => void
+  setupWorkspace: () => void
+}
+
+const OfferToInstall: React.FC<{
+  children: React.ReactNode
+  pythonBinPath: string
+  installDvc: () => void
+}> = ({ installDvc, pythonBinPath, children }) => (
+  <div>
+    <p>
+      {`DVC & DVCLive can be auto-installed as packages with ${pythonBinPath}`}
```

Comment:
Technically, I don't think we have to wrap the entire string with curly braces: 
```suggestion
      DVC & DVCLive can be auto-installed as packages with {pythonBinPath}
```